src/components/MovieListBookmarks.js

```diff
@@ -0,0 +1,97 @@
+import Link from 'next/link';
+import styled from 'styled-components';
+
+import useStore from '../hooks/useStore';
+
+/* Used pic width's w92 w154 & w185 */
+const Article = styled.article`
+	position: relative;
+	max-width: 156px;
+	padding: 0;
+	animation: fadeIn 1s ${({delay}) => delay}s forwards;
+	border: 1px solid rgba(0, 0, 0, 0.25);
+	border-radius: 16px;
+	opacity: 0;
+	background-color: #fff;
+	box-shadow: 1px 1px 10px 5px rgba(0, 0, 0, 0.2);
+	color: #fff;
+	@keyframes fadeIn {
+		0% {
+			opacity: 0;
+		}
+		100% {
+			opacity: 1;
+		}
+	}
+	& button {
+		position: absolute;
+		right: 0;
+		padding: 6px 8px;
+		border: none;
+		border-radius: 0 14px 0 0;
+		background-color: white;
+		&:hover {
+			background-color: #f30;
+			color: white;
+		}
+	}
+`;
+
+const Picture = styled.img`
+	max-width: 154px;
+	border-radius: 16px 16px 0 0;
+`;
+
+const Container = styled.div`
+	display: flex;
+	flex-wrap: wrap;
+	padding: 16px;
+	gap: 16px;
+`;
+
+const MovieHeadline = styled.p`
+	min-height: 56px;
+	padding: 12px 8px 16px 12px;
+	color: black;
+	overflow-wrap: break-word;
+`;
+
+export default function MovieListBookmarks(props) {
+	const urlSource = props.type === 'tv' ? '/tv/' : '/movie/';
+	let arrData = props.data;
```

Comment:
I highly recommend to destructure props in terms of more readable and clear code, especially when passing down props in React
```suggestion
export default function MovieListBookmarks({data, type}) {
	const urlSource = type === 'tv' ? '/tv/' : '/movie/';
	let arrData = data;
```

Reply:
done